### PR TITLE
Fixed allows vs allow on Smart contracts page

### PR DESCRIPTION
**Pull request by tymtam2**
## Description

There is a plural/singular error in the page. 

They are computer programs stored on the blockchain that allow\*s\* us to convert traditional contracts into digital parallels.

I changed it to

 They are computer programs stored on the blockchain that allow converting traditional contracts into digital parallels.

### CODE REVIEW

File: src/content/smart-contracts/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar: true
 
 # Introduction to smart contracts {#introduction-to-smart-contracts}
 
-Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). They are computer programs stored on the blockchain that allows us to convert traditional contracts into digital parallels. Smart contracts are very logical - following an if this then that structure. This means they behave exactly as programmed and cannot be changed.
+Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). They are computer programs stored on the blockchain that allow converting traditional contracts into digital parallels. Smart contracts are very logical - following an if this then that structure. This means they behave exactly as programmed and cannot be changed.
```
Comment (minimalsm) [inline src/content/smart-contracts/index.md:10]: Nice catch :-) we are modifying the plural noun "computer programs", so this should be singular.

However, the other change (us to convert -> converting) makes the tone less friendly, and I think we should revert it. 

```suggestion
Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). They are computer programs stored on the blockchain that allow us to convert traditional contracts into digital parallels. Smart contracts are very logical - following an if this then that structure. This means they behave exactly as programmed and cannot be changed.
```

**Review by corwintines — APPROVED** [review]
Thanks, @tymtam2 :)

**Comment by corwintines** [comment]
@all-contributors can you add @tymtam2 for docs

**Comment by corwintines** [comment]
Hey @tymtam2 , thanks for the PR :)

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also!

**Pull request merged**